src/formatDate.js

```diff
@@ -61,28 +61,27 @@ function formatDate(date, fromFormat, toFormat) {
   }
 
   for (let i = 0; i < toFormat.length - 1; i++) {
-    if (toFormat[i] === 'DD') {
-      result.push(dateParts['DD']);
-      continue;
-    }
-
-    if (toFormat[i] === 'MM') {
-      result.push(dateParts['MM']);
-      continue;
-    }
+    switch (true) {
+      case toFormat[i] === 'DD':
```

Comment:
```suggestion
    switch (toFormat[i]) {
      case 'DD':
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
unfortunally i cant change it, because then my third case brakes
maybe you can give me a hint how i can do this?

Reply:
Add two more cases: 'YY' and 'YYYY':
```js
case 1:
case 2:
  // your code for both
```